datafusion/core/tests/parquet/filter_pushdown.rs

```diff
@@ -80,15 +81,15 @@ async fn single_file() {
         // request_method = 'GET'
         .with_filter(col("request_method").eq(lit("GET")))
         .with_pushdown_expected(PushdownExpected::Some)
-        .with_expected_rows(688);
+        .with_expected_rows(745);
```

Comment:
@alamb https://github.com/apache/arrow-datafusion/blob/72fb87e27bfaf62580f30fb6e85aed2a22fbe744/test-utils/src/data_gen.rs#L172 I think here every running test should get diff result 🤔 , but from the fact only change the generate method this would change

Reply:
I am not quite sure what you are asking here @Ted-Jiang . It seems like most of the tests in this file have been updated as well